src/Template/Bake/Model/table.ctp

```diff
@@ -119,7 +119,7 @@ foreach ($validation as $field => $rules):
                     $field
                 );
                 $validationMethods[] = sprintf(
-                    "->notEmpty('%s')",
+                    "->notBlank('%s')",
```

Comment:
Err disregard my above comment. This `notEmpty()` is `Validator::notEmpty()` not `Validation::notEmpty()` so no change needs to be done.